.env

```diff
@@ -0,0 +1,7 @@
+# This file defines environment variables we want to be set in development
+# environments.  Commands run via Just should have these set already but we
+# want to support other invocation mechanisms. We use a `.env` file as VSCode
+# should pick these up by default, see:
```

Comment:
I can see myself using the terminal in VS Code and wondering why it isn't picking up the environment variables defined within `.env`. Of course, that probably wouldn't happen for _this_ environment variable, because I'd use `just test`. But I can see `.env` growing.

```suggestion
# want to support other invocation mechanisms. We use a `.env` file as VSCode's
# Python extension (but not the terminal) should pick these up by default, see:
```

Reply:
Oh thanks! I hadn't realised this so it's a useful clarification.